modules/cookbooks/pages/kafka_migrator.adoc

```diff
@@ -0,0 +1,633 @@
+      KAFKA_CFG_AUTHORIZER_CLASS_NAME: "org.apache.kafka.metadata.authorizer.StandardAuthorizer"
+      KAFKA_CFG_SUPER_USERS: User:redpanda;User:ANONYMOUS
+    ports:
+      - 9092:9092
+      - 19092:19092
+    healthcheck:
+      test: [ "CMD", "kafka-topics.sh", "--bootstrap-server=localhost:9092", "--list" ]
+      start_period: 5s
+      interval: 3s
+
+  init_source:
+    image: bitnami/kafka
+    working_dir: /opt/bitnami/kafka/bin
+    entrypoint: /bin/bash
+    depends_on:
+      source:
+        condition: service_healthy
+    command: >
+      -c "kafka-topics.sh --bootstrap-server source:19092 --create --if-not-exists --topic foo --replication-factor=1 --partitions=2 &&
+          kafka-topics.sh --bootstrap-server source:19092 --create --if-not-exists --topic bar --replication-factor=1 --partitions=2 &&
+          echo 'Created topics:' &&
+          kafka-topics.sh --bootstrap-server source:19092 --list --exclude-internal &&
+          kafka-acls.sh --bootstrap-server source:19092 --add --allow-principal User:redpanda --operation Read --topic foo &&
+          kafka-acls.sh --bootstrap-server source:19092 --add --deny-principal User:redpanda --operation Read --topic bar
+          echo 'Created ACLs:' &&
+          kafka-acls.sh --bootstrap-server source:19092 --list"
+
+  source_schema_registry:
+    image: bitnami/schema-registry
+    environment:
+      SCHEMA_REGISTRY_KAFKA_BROKERS: PLAINTEXT://source:19092
+    ports:
+      - 8081:8081
+    depends_on:
+      source:
+        condition: service_healthy
+
+  destination:
+    image: redpandadata/redpanda
+    command:
+      - redpanda
+      - start
+      - --node-id 0
+      - --mode dev-container
+      - --set rpk.additional_start_flags=[--reactor-backend=epoll]
+      - --set redpanda.auto_create_topics_enabled=false
+      - --kafka-addr 0.0.0.0:9093
+      - --advertise-kafka-addr localhost:9093
+      - --schema-registry-addr 0.0.0.0:8081
+    ports:
+      - 8082:8081
+      - 9093:9093
+      - 9645:9644
+----
+
+[source,console]
+----
+docker-compose -f docker-compose.yml up --force-recreate -V
+----
+
+NOTE: This command uses an `init` container to create two topics, `foo` and `bar`, each with two partitions and an associated ACL.
+
+endif::[]
+
+ifdef::env-cloud[]
+
+== Create a Kafka cluster and a Redpanda Cloud cluster
+
+First, you'll need to provision two clusters, a Kafka one called `source` and a Redpanda Cloud one called `destination`. We'll use the following sample connection details throughout the rest of this cookbook:
+
+`source`
+----
+broker:          source.cloud.kafka.com:9092
+schema registry: https://schema-registry-source.cloud.kafka.com:30081
+username:        kafka
+password:        testpass
+----
+
+`destination`
+----
+broker:          destination.cloud.redpanda.com:9092
+schema registry: https://schema-registry-destination.cloud.redpanda.com:30081
+username:        redpanda
+password:        testpass
+----
```

Comment:
Wasn't sure if this is the right way to format this section. Happy to change it if need be.